rosdep/base.yaml

```diff
@@ -2408,6 +2411,18 @@ libgstreamer1.0-dev:
   fedora: [gstreamer1-devel]
   gentoo: ['media-libs/gstreamer:1.0']
   ubuntu: [libgstreamer1.0-dev]
+libgstrtspserver-1.0-0:
+  debian: [libgstrtspserver-1.0-0]
+  ubuntu: [libgstrtspserver-1.0-0]
+libgstrtspserver-1.0-0-dbg:
+  debian: [libgstrtspserver-1.0-0-dbg]
+  ubuntu: [libgstrtspserver-1.0-0-dbg]
+libgstrtspserver-1.0-dev:
+  debian: [libgstrtspserver-1.0-dev]
+  ubuntu: [libgstrtspserver-1.0-dev]
+libgstrtspserver-1.0-doc:
+  debian: [libgstrtspserver-1.0-doc]
+  ubuntu: [libgstrtspserver-1.0-doc]
```

Comment:
Same goes for documentation packages; it's just not something that we'll need to build or on the buildfarm.  So please remove this.

Reply:
Also makes sense.